meetings/tool/src/main/java/org/sakaiproject/meetings/controller/MeetingsController.java

```diff
@@ -606,6 +606,16 @@ public String getI18nProperties(@PathVariable String bundle, @PathVariable Strin
         return i18n.toString();
     }
     
+    /**
+     * Returns true if MS Teams is set up in Sakai properties
+     * @return
+     * @throws MeetingsException
+     */
+    @GetMapping(value = "/meetings/isteamssetup", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I'm not an expert on API definition but maybe /meetings/teams/status is better?

Reply:
Yeah, that makes more sense to me, I'm gonna change it